ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+            sh:description "Traffic direction, i.e. right-hand or left-hand traffic"@en ;
+            sh:in ("left-hand" "right-hand") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of trafficDirection failed!"@en ;
+            sh:name "trafficDirection"@en ;
+            sh:order 3 ;
+            sh:path ositrace:trafficDirection ],
+        [ skos:example "object list, detection list" ;
+            sh:datatype xsd:string ;
+            sh:description "Level of granularity of sensor data"@en ;
+            sh:in ("object list" "detection list") ;
+            sh:message "Validation of granularity failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "granularity"@en ;
+            sh:order 4 ;
+            sh:path ositrace:granularity ],
+        [ skos:example "119" ;
+            sh:datatype xsd:string ;
+            sh:description "Identifier of scenario performed in the trace file"@en ;
+            sh:message "Validation of scenarioIdentifier failed!"@en ;
+            sh:name "scenarioIdentifier"@en ;
+            sh:order 5 ;
+            sh:path ositrace:scenarioIdentifier ],
+        [ skos:example "2024-05-11T13:20:03.250320Z" ;
+            sh:datatype xsd:dateTimeStamp ;
+            sh:description "Exact start timestamp of the recorded trace"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of startTime failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "startTime"@en ;
+            sh:order 6 ;
+            sh:path ositrace:startTime ],
+        [ skos:example "2024-05-11T13:20:05.250120Z" ;
+            sh:datatype xsd:dateTimeStamp ;
+            sh:description "Exact stop timestamp of the recorded trace"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of stopTime failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "stopTime"@en ;
+            sh:order 7 ;
+            sh:path ositrace:stopTime ],
+        [ sh:description "Host moving object in trace file"@en ;
+            sh:maxCount 1 ;
+            sh:name "hostMovingObject"@en ;
+            sh:node ositrace:MovingObjectShape ;
+            sh:order 8 ;
+            sh:path ositrace:hostMovingObject ],
+        [ sh:description "Target moving object in trace file"@en ;
+            sh:name "targetMovingObject"@en ;
+            sh:node ositrace:MovingObjectShape ;
+            sh:order 9 ;
+            sh:path ositrace:targetMovingObject ],
+        [ sh:description "Description of event of interest in trace file"@en ;
+            sh:name "event"@en ;
+            sh:node ositrace:EventShape ;
+            sh:order 10 ;
+            sh:path ositrace:event ] ;
+    sh:targetClass ositrace:Content .
+
+ositrace:DataSourceShape a sh:NodeShape ;
+    sh:property [ skos:example "PMSF DroneTracker; ARS 548 Sensor Output" ;
+            sh:datatype xsd:string ;
+            sh:description "Main acquisition device"@en ;
+            sh:maxCount 1 ;
```

Comment:
Shouldn't it be possible to state also multiple measurement systems? I think in one OSI file could be also multiple sensors be integrated.

Reply:
It could be used for anything, however practically speaking I do not think making this machine interpretable is the most important thing. I tried to stay close to hdmap on this, which might also have multiple acquisition devices if stuff gets merged. If we want to evolve on this, I would steer away from the idea of a main acquisition device, and then multiple entries would make sense.